src/applications/gi/components/profile/HeadingSummary.jsx

```diff
@@ -35,9 +35,15 @@ class HeadingSummary extends React.Component {
       <div className="heading row">
         <div className="usa-width-two-thirds medium-8 small-12 column">
           <h1>{it.name}</h1>
+          {__BUILDTYPE__ !== 'production' && (<AlertBox
+            content={(<p>Are you enrolled in this school? <a href="https://www.benefits.va.gov/GIBILL/FGIB/Restoration.asp" onClick={this.props.onViewWarnings}>Find out how the VA can restore your benefits</a></p>)}
```

Comment:
It seems odd that this is a link with both an onClick and an external href. It also may need to open in a new tab.